package.json

```diff
@@ -96,7 +96,8 @@
     "vinyl-source-stream": "^2.0.0"
   },
   "overrides": {
-    "glob-parent": "6.0.2"
+    "glob-parent": "6.0.2",
+    "postcss": "^8.4.31"
```

Comment:
> **Note**
> I tried to reference a value of `$postcss` as described in the [npmjs overrides docs](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides), but could not get it to recognize the reference. Not sure what I am doing wrong. It should be ok to move forward without the reference since we are viewing this override as a temporary fix and we hopefully won't have to update the postcss devDependency version before that, but might be a good thing to figure out for future overrides like this. 

```suggestion
    "postcss": "$postcss"
```